ui/app/pages/first-time-flow/create-password/import-with-seed-phrase/import-with-seed-phrase.component.js

```diff
@@ -296,10 +296,9 @@ export default class ImportWithSeedPhrase extends PureComponent {
           </span>
         </div>
         <Button
-          type="primary"
+          type="submit"
```

Comment:
It doesn't seem ideal to me to re-use the existing `type` prop for a new purpose. The existing `type` prop seems to be for styling the button, rather than for specifying the `type` attribute to use for the button.

Ideally this `Button` wrapper would use a different prop for styling, and pass `type` on to `button` via `buttonProps` with everything else, but... that seems like a larger change. I mean feel free to look into doing that if you want, but we could in the meantime just add a new `buttonType` or `submit` prop to disambiguate these two uses.

Reply:
ah, good point! adding a `submit` `bool` prop makes a lot more sense